.circleci/config.yml

```diff
@@ -6,18 +6,29 @@ jobs:
       - image: "cimg/elixir:1.13.3"
     steps:
       - checkout
+      - run: mix local.hex --force
+      - run: mix deps.get
       - run: mix run
```

Comment:
Curious about why we're running `mis run` here. Would it make sense to run `mix compile` as the build step instead of actually running it?